### PR TITLE
Add parsing of timing data and NELM, NSW

### PR DESCRIPTION
**Pull request by zhubonan**
Timing data can be very useful for benchmarks, convergence
testings.
The extraction of NELM/NSW is needed for determining whether
the electronic/ionic convergence is reached since vasp always
pretends they are converged once the maximum step is reached.

### CODE REVIEW

File: parsevasp/vasprun.py [hunk 1]
```diff
@@ -1130,6 +1134,67 @@ def _fetch_sigmaw(self, xml):
 
         return sigma
 
+    def _fetch_nelmw(self, xml):
```
Comment (espenfl) [inline parsevasp/vasprun.py:1137]: Thanks a lot. Could you also add code to the event driven extraction? We need that for the bigger files which we can not load into memory.

**Review by espenfl — COMMENTED** [review]
@zhubonan Greatly appreciated. Thanks a lot. Just a comment regarding implementation of the extraction of the INCAR tags in the event parser as well.

File: parsevasp/vasprun.py (hunk 1 above)
Comment (zhubonan) [reply parsevasp/vasprun.py:1148]: Done!

**Comment by zhubonan** [comment]
@espenfl Can you have a look at this one? the last commit has implemented the event parser as well.

**Comment by espenfl** [comment]
Awesome @zhubonan. Thanks a lot for the contribution. Merging now.

**Pull request merged**